semperfiwebdesign/all-in-one-seo-pack
# Change JQuery CDN to Microsoft

Issue #2607

## Proposed changes

Changes the CDN we use for JQuery UI to Microsoft so Chinese users are able to fetch the CSS file.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

Connect to a Chinese VPN server with a free or paid service like ExpressVPN and check whether the JQuery tabs are still styled and that there aren't any loading issues.
